internal/providers/file/file.go

```diff
@@ -31,6 +32,9 @@ const (
 
 func FetchConfig(f resource.Fetcher) (types.Config, report.Report, error) {
 	filename := os.Getenv(cfgFilenameEnvVar)
+	if f := flag.Lookup("validate"); f != nil {
```

Comment:
This feels gross, but the alternatives are changing the interfaces to allow passing more information to FetchConfig, using a global, or changing the enviroment variables. 

Also, this has a bug (flag.Lookup does not work as I thought it did), fixing.